doc/source/actions.rst

```diff
@@ -357,6 +357,13 @@ The ``merge`` action merges the pull request into its base branch. The
        Note that the ``rebase`` method has many drawbacks, see :ref:`strict
        rebase`.
 
+   * - ``priority``
+     - positive integer or one of ``low``, ``default``, ``high``
+     - `default`
+     - This set the priority of the pull request in the queue when ``smart``
+       :ref:`strict merge` is enabled. Higher number get merged first.
+       ``low``, ``default``, ``high`` are aliases for ``0``, ``10``, ``100``
```

Comment:
```suggestion
     - Sets the priority of the pull request in the queue when ``smart``
       :ref:`strict merge` is enabled. Higher numbers get merged first.
       ``low``, ``medium``, ``high`` are aliases for ``0``, ``10``, ``100``
```

I'm a bit disappointed you did not use emoji for this. Can we add some? 

- 🐌
- 🐢
- 🐇
- 🚲
- 🏎
- 🚀

🤣